src/JHipsterNetSampleApplication/Data/Extensions/DbSetExtensions.cs

```diff
@@ -33,5 +34,63 @@ private static PropertyInfo GetKeyProperty(Type type)
             //TODO complete with FluentAPi
             return null;
         }
-    }
+
+        public static void RemoveNavigationProperty<TEntity, TOwnerEntity>(this DbSet<TEntity> receiver, TOwnerEntity ownerEntity, object id)
+            where TEntity : class
+            where TOwnerEntity : class
+        {
+            bool success;
+            var receiverObjects = receiver.ApplyWhere(ownerEntity.GetType().Name + "Id", id, out success);
```

Comment:
Can we avoid the `out` usage? It's tricky and error prone. I'm not a huge fan but you can still try to convince me 😄

Reply:
I used try catch and exceptions throws and tried to adding logs but I'm not really sure of how works Serilog. I will PR again, can you tell me if it is right ?